src/test/java/com/exasol/adapter/dynamodb/mapping/reader/ColumnMappingReaderTest.java

```diff
@@ -0,0 +1,132 @@
+package com.exasol.adapter.dynamodb.mapping.reader;
+
+import static com.exasol.adapter.dynamodb.mapping.ToJsonPropertyToColumnMapping.OverflowBehaviour.NULL;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.adapter.dynamodb.documentpath.DocumentPathExpression;
+import com.exasol.adapter.dynamodb.mapping.LookupFailBehaviour;
+import com.exasol.adapter.dynamodb.mapping.ToJsonPropertyToColumnMapping;
+import com.exasol.adapter.dynamodb.mapping.ToStringPropertyToColumnMapping;
+
+class ColumnMappingReaderTest {
+
+    /**
+     * Tests for the {@link ColumnMappingReader} uses the correct default values for the
+     * {@link ToStringPropertyToColumnMapping}.
+     */
+    @Test
+    void testToStringColumnDefaultValues() {
+        assertToStringDefinitionDefaultValues(Json.createObjectBuilder().build());
+    }
+
+    @Test
+    void testToStringColumnExplicitDefaultValues() {
+        final JsonObject definition = Json.createObjectBuilder()//
+                .add("maxLength", 254)//
+                .add("overflow", "TRUNCATE")//
+                .add("destName", "test")//
+                .add("required", false).build();
+        assertToStringDefinitionDefaultValues(definition);
+    }
+
+    void assertToStringDefinitionDefaultValues(final JsonObject definition) {
+        final ToStringPropertyToColumnMapping columnMapping = (ToStringPropertyToColumnMapping) ColumnMappingReader
+                .getInstance()
+                .readColumnMapping("toStringMapping", definition, new DocumentPathExpression.Builder(), "test", false);
+        assertAll(// )
```

Comment:
Do you need `)` here?

Reply:
it's closed in line 51

Reply:
I mean you have a `)` in the comment. Looks like a typo